.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 88
+extend-ignore = E203, W503
+max_complexity = 32
```

Comment:
Setting should be added to `pyproject.toml`

Reply:
See https://pypi.org/project/Flake8-pyproject/

Reply:
As of 6.0 this is not a documented configuration location for fake8: 
https://flake8.pycqa.org/en/6.0.0/user/configuration.html

Is there some more updated documentation of the feature of using the file for configuration?